runtests

```diff
@@ -5,8 +5,10 @@ set -o pipefail
 
 main() {
   # abort if there are uncommitted local changes
-  msg="You have uncommitted changes. Not proceeding, as this script changes some files in place."
-  git diff-index --quiet HEAD -- || (echo "$msg" && exit 1)
+  if [[ "$1" != "--force" ]]; then
```

Comment:
Instead of checking `$1`, which is also used below for `--env <env-file>`, what about `if [[ "${*}" =~ --force ]]; then` to see if `--force` is anywhere in args?